src/test/scala/com/cognite/sdk/scala/v1/TransformationsTest.scala

```diff
@@ -0,0 +1,214 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+package com.cognite.sdk.scala.v1
+
+import cats.Id
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import com.cognite.sdk.scala.common._
+import com.cognite.sdk.scala.v1.FlatOidcCredentials._
+
+import java.time.temporal.ChronoUnit
+import java.time.Instant
+import scala.concurrent.duration.DurationInt
+import scala.util.control.NonFatal
+import java.util.UUID
+
+@SuppressWarnings(
+  Array(
+    "org.wartremover.warts.NonUnitStatements",
+    "org.wartremover.warts.SizeIs",
+    "org.wartremover.warts.AsInstanceOf",
+    "org.wartremover.warts.OptionPartial",
+    "org.wartremover.warts.StringPlusAny"
+  )
+)
+class TransformationsTest extends CommonDataModelTestHelper with RetryWhile {
+
+  private val auth = OAuth2
+    .ClientCredentialsProvider[IO](credentials)
+    .unsafeRunTimed(1.second)
+    .get
+    .getAuth
+    .unsafeRunSync()
+    .asInstanceOf[OidcTokenAuth]
+
+  private lazy val client = new GenericClient[Id](
+    "scala-sdk-test",
+    "extractor-bluefield-testing",
+    "https://bluefield.cognitedata.com",
+    BearerTokenAuth(auth.bearerToken),
+    None,
+    None
+  )
+
+  def shortRandomUUID(): String = UUID.randomUUID().toString.substring(0, 8)
```

Comment:
Please use this for all the identifiers in items you create, otherwise tests can't be run concurrently.

We've had flaky tests before because of this which are fixed in commits like https://github.com/cognitedata/cognite-sdk-scala/pull/468/files#diff-6a059987fdd28e9fe1eaa0c90ab2bea6ff8ec2894a2b559ddb0a630107d2c9f9R101 and https://github.com/cognitedata/cognite-sdk-scala/commit/287507037cbe1e25cbce5ff3ae9de80bd6d02158, so let's not create more of them.